src/App.js

```diff
@@ -76,41 +80,78 @@ class App extends Component {
               showCheckbox={false}
               className="authors"
               innerClass={{
-                "list": "author-list"
+                list: "author-list"
               }}
               placeholder="Filter by author name"
               filterLabel="Authors"
             />
           </div>
           <div className={"mainBar"}>
             <SelectedFilters />
-            <ResultCard
-              componentId="results"
+            <ReactiveList
+              componentId="SearchResult"
               dataField="original_title"
+              size={10}
```

Comment:
Let's keep the size as `8` because each row consists of 4 cards so it would look good.